cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -204,7 +203,6 @@ static int _indication_rx(ieee802154_dev_t *dev, void *buf, size_t size, ieee802
         res = 0;
     }
 
-    RFCORE_SFR_RFST = ISFLUSHRX;
```

Comment:
Why was this one removed? Without the flush I'm under the impression that `rfcore_peek_rx_fifo(0);` will not always point to the `CRC` since the `peek` is not relative to the FIFO pointer, or not?

Reply:
I do understand that this follows:

```
The read function doesn't need semantics for dropping frames, since set_trx_state(RX_ON) already flushes the framebuffer. this simplifies the driver implementation and reduces potential bugs
```

I just want to check this doesn't affect the crc peek part.

Reply:
Ahh I think I understand, know after RX it does not switch back to RX immediately but only if that state is requested right?

Reply:
> Ahh I think I understand, know after RX it does not switch back to RX immediately but only if that state is requested right?

Exactly :)